### PR TITLE
feat: Write the basic settings of docker

### PR DESCRIPTION
**Pull request by StiVit**

### CODE REVIEW

File: app/database.py [hunk 1]
```diff
@@ -3,8 +3,9 @@
 from sqlalchemy.orm import sessionmaker
 
 # Database configurations
-SQLALCHEMY_DATABASE_URL = "sqlite:///./test.db"
-engine = create_engine(SQLALCHEMY_DATABASE_URL, connect_args={"check_same_thread": False})
+SQLALCHEMY_DATABASE_URL = "postgresql://sage:sage@localhost:5432/sage"
```
Comment (grumpycatyo-collab) [inline app/database.py]: I think you need to change the credentials here. Use something proper for your project

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+services:
+  postgres:
+    image: postgres:14-alpine
+    ports:
+      - 5432:5432
+    volumes:
+      - postgres-data:/data/postgres
+    environment:
+      - POSTGRES_PASSWORD=sage
```
Comment (grumpycatyo-collab) [inline docker/docker-compose.yml]: See my previous comment about credentials

Comment (grumpycatyo-collab) [inline docker/docker-compose.yml:11]: See my previous comment about credentials

File: app/models/models.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# app/models.py
+
+from sqlalchemy import Column, Integer, String, Date, Double, ForeignKey
+from app.database import Base
+from enumerations.region import Region
+from enumerations.payment_method import PaymentMethod
+from enumerations.product_category import ProductCategory
+
+# SQLAlchemy models
+class Transaction(Base):
```
Comment (grumpycatyo-collab) [inline app/models/models.py:11]: Nice!

**Review by grumpycatyo-collab — APPROVED** [review]
Overall is pretty good. You just need to make some minor changes (tho I am approving)
Next PR I guess would involve making the algorithm for mapping the dataset to the db

**Comment by StiVit** [comment]
Gonna make the endpoints for the new crud implemented

**Pull request merged**